### ISSUE TITLE
Cay_C_EE_PVE and Mnt_C_EE have incorrect positions after old json is loaded in new version of editor

### ISSUE THREAD

**Issue by thor79** [GrapeshotGames/ServerGridEditor#73]
Upon downloading the new version of the Server Grid Editor tonight after the PTR launched, I tried loading the 5x8 map json we currently use on our unofficial server and immediately saved it to compare the json files directly (done with Visual Studio Code and the extension compareIt). The result is with these two islands, which each have a single instance in the map, their positions changed on the map. They share the same shape and they moved in the same direction based on their rotation differences. The only differences in the json for those islands are the addition of "island points" (no doubt plays a factor in PVE) and the island Width and Height. The WorldX and WorldY positions did not change in the JSON file.

As we currently use this json on our existing map, this makes it difficult to use this new version. Our playerbase largely does not want a wipe like the official servers are doing, so we are doing our best to add the new content without modifying the positions of any existing islands.

**Comment by smooreace** [comment]
This has always been an issue. The simple solution is to simply pull the
new lines from the newly created json and add just those to your existing
json. It's fairly obvious that GS doesn't care about the editor enough to
make it a finished product. So all we can do is learn to work around their
inability to produce a properly functioning product

**Comment by thor79** [comment]
Yes, I saw that others had the same issue with older versions. This was not an issue for me until this version though. This is why I opened the issue...same symtoms, unique cause with the new version. 

The source templates changed...specifically the x and y values and the addition of the island points. the x and y changed for more islands than just those above, but they did not change for all islands. Other islands that also had their x and y changed did not move in the editor.

And I tried your "simple solution"...no go, they still move.

It won't surprise me if they don't give a damn how they affect unofficial servers trying to build and maintain their own maps.

**Comment by smooreace** [comment]
Nothing moves in the 'simple solution' because you don't transfer that
data.

Open the original servergrid file and make your changes, island additions
and removals, and export as usual. Then copy only the data directly related
to those specific changes to the proper locations in the old exported
servergrid file. Launch, and prosper. Because of the known issues with
moving islands, no one should ever be using re-exported files. It's just
begging for issues. I've been manipulating my map, expanded 4 times, and
several other significant changes since just after release this way. Works
flawlessly. Just have to be careful of what you move, and where you put it.
Realistically, the grid editor isn't needed at all! But writing a config
100% manually would probably be one hell of a daunting task!!

And... my response wasnt to poke at your starting a new issue. Yours
actually has quality data in it! It was just to assist in getting around
those issues until GS decides that fixing the problem is important (good
luck!).

**Comment by thor79** [comment]
The island moved already. Our Live map has the islands moved (no changes have been made to that json since before the large patch). I'm verifying right now if anyone actually settled on the islands that moved.
